packages/x-components/src/utils/__tests__/normalize.spec.ts

```diff
@@ -25,6 +25,8 @@ describe('testing string normalization', () => {
     it('trims and lowercases characters', () => {
       /* eslint-disable @typescript-eslint/unbound-method */
       const originalStringNormalize = String.prototype.normalize;
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-expect-error TS does not allow removing properties that are not optional.
```

Comment:
Obviously, we shouldn't remove a property which is not optional. TS now detects this errors and complains about them. Again this is for a test that is testing exactly what happens if there is no `normalize` function available in strings. AFAIK this function is not available only in IE, which isn't a problem as we don't support it, and in Android webviews, which are a problem.

Can somebody with an Android device can try to check it using a browser like the Facebook one? I think this was one of the apps that included old webviews.

Reply:
Good to know. Fortunately, I don't use android devices nor use facebook :trollface: